DeskReserve/Domain/FloorService.cs

```diff
@@ -0,0 +1,67 @@
+﻿using DeskReserve.Data.DBContext.Entity;
+using DeskReserve.Data.DBContext;
+using Microsoft.EntityFrameworkCore;
+
+namespace DeskReserve.Domain
+{
+    public class FloorService
+    {
+        private readonly ApplicationDbContext _context;
+
+        public FloorService(ApplicationDbContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<List<Floor>> GetFloors()
+        {
+            return await _context.Floors.ToListAsync();
+        }
+
+        public async Task<Floor> GetFloor(Guid id)
+        {
+            return await _context.Floors.FindAsync(id);
+        }
+
+        public async Task<bool> UpdateFloor(Guid id, Floor floor)
+        {
+            if (id != floor.FloorId)
```

Comment:
?

Reply:
What if the id in floor object provided in the request body contains different id form the one in the route?